DevREADME.md

```diff
@@ -99,4 +99,132 @@ You may encounter the failure of npm install, the error message would say:
 Error installing vscode.d.ts: Error: read ECONNRESET
 If you encounter this problem, maybe your NODE is too old, in version v10.13.0 it 
 can work well.
-or run "npm cache clean -force"
\ No newline at end of file
+or run "npm cache clean -force"
+
+---
+
+## Style Guide
+Below are some general code guidelines that should be followed when adding or updating the code base. It is the expectation that participating members actively apply this style guide to legacy code during maintenance so we can eventually have a noticeably consistent developer experience.
+### Names
+1. Use `PascalCase` for type/interface names, enums, setters, getters and global singletons
+2. Use `camelCase` for function, argument and local variable names.
```

Comment:
Might be a good idea these into 3 statements:
- Use `PascalCase` for type/interface names, enums, and global singleton instance variables
- Use `PascalCase` for getters and setters within singleton classes
- Use `cabelCase` for functions, getters, setters, arguments and local variables